data_scraping/tensorflow_data.py

```diff
@@ -273,11 +273,12 @@ def main(startDate, endDate, dataFolderPath):
     for url in tqdm(urls):
         startTimeForUrl = time.time()
         get_data_from_repository(url, driver, startTimeForUrl, dataFolderPath)
-    endTime = time.time()
-    print("Total time taken:", {endTime - startTime})
+        endTime = time.time()
+        print("Total time taken:", {endTime - startTime})
+    print("Finished")
 
 
-startDate = sys.argv[1]
-endDate = sys.argv[2]
-dataFolderPath = sys.argv[3]
-main(startDate, endDate, dataFolderPath)
+# startDate = sys.argv[1]
+# endDate = sys.argv[2]
+# dataFolderPath = sys.argv[3]
+# main(startDate, endDate, dataFolderPath)
```

Comment:
We can keep both approaches right?

```
if __name___ == "__main__":
    startDate....
```

Reply:
if we use the commented one the file can't be imported for testing purposes

Reply:
And that's why we use the above-mentioned approach... 
So when the file imported then it won't run the file but when I want to run the file using `py filename.py` then it will :)

Reply:
Yeah I have created an issue for the same